### DISCUSSION TITLE
How to enable EnableMultiWindows on Android Appium

### DISCUSSION THREAD

**Discussion by rohith-yml** [serenity-bdd/serenity-core#2879] [General]
Hello , 
I have a use case where I need to access the MultiWindow feature and I am unable to do it.

added the below property in .conf
**appium.enableMultiWindows = true**

Got below error 
[WD Proxy] Got response with status 500: {"sessionId":null,"value":{"error":"session not created","message":"io.appium.uiautomator2.common.exceptions.InvalidArgumentException: Invalid '**enableMultiWindows**' setting value type. Got: java.lang.String. Expected: java.lang.Boolean.","stacktrace":"io.appium.uiautomator2.common.exceptions.SessionNotCreatedException: **io.appium.uiautomator2.common.exceptions.InvalidArgumentException: Invalid 'enableMultiWindows' setting value type. Got: java.lang.String. Expected: java.lang.Boolean.\n\tat** io.appium.uiautomator2.handler.NewSession.safeHandle(NewSession.java:58)\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandle...

@wakaleo

**Comment by gouri-17** [comment]
@rohith-yml Did you find any solution for this?
